vxfreeswitch/voice.py

```diff
@@ -232,11 +282,23 @@ def setup_transport(self):
         def protocol():
             return FreeSwitchESLProtocol(self)
 
+        def client_protocol():
+            return FreeSwitchESLClientProtocol(self)
+
         factory = ServerFactory()
         factory.protocol = protocol
-
         self.voice_server = yield self.config.twisted_endpoint.listen(factory)
 
+        self.client_factory = ClientFactory()
+        self.client_factory.protocol = client_protocol
+
+    @inlineCallbacks
+    def create_voice_client(self):
```

Comment:
Perhaps we should rename this function to `create_outbound_client` or `create_dialer_client` since everything in this package is to do with voice?